generate-adjudicator-bindings.sh

```diff
@@ -0,0 +1,6 @@
+set -e
```

Comment:
When I pulled down this feature branch, `generate-adjudicator-bindings.sh` is not executable by default

Reply:
Works fine for me though.

Reply:
I think this means you need to explicitly run `sh generate-adjudicator-bindings.sh`? Happy to make it executable if you think that makes sense @kerzhner ?